src/react/components/buttons/FaucetButton.tsx

```diff
@@ -0,0 +1,16 @@
+import "../../../styles/navbar.scss";
+import { CredixButton } from "@components/buttons/CredixButton";
+import React from "react";
+
+interface Props {
+	text: string;
+	className?: string;
+}
+
+export const FaucetButton = (props: Props) => {
```

Comment:
is this used somewhere?

Reply:
yes, on the help page and in the navbar

Reply:
I checked out the branch and I'm getting `Unused constant FaucetButton`, I think there are 2 of them, one in components and one in components/buttons